scripts/sct_label_utils.py

```diff
@@ -125,6 +125,10 @@ def process(self, type_process):
             self.output_image = self.continuous_vertebral_levels()
         if type_process == 'create-viewer':
             self.output_image = self.launch_sagittal_viewer(self.value)
+        if type_process == 'remove':
+            self.output_image = self.remove_labels(self.value)
+        if type_process == 'keep':
+            self.output_image = self.remove_labels(self.value,isKeep = True)
```

Comment:
the name of the function is not intuitive (we expect it to remove labels, not to keep them). I suggest to rename it `remove_or_keep_labels(self.value, action={'keep', 'remove'})`

Reply:
Got it. Changes applied in commit [ec624a5](https://github.com/neuropoly/spinalcordtoolbox/commit/ec624a5d324d90bcddf03897b78c861cb55abc82)